packages/core/react/use-chat.ts

```diff
@@ -159,10 +159,38 @@ const getStreamedResponse = async (
 
   if (isComplexMode) {
     while (true) {
-      const { done, value } = await reader.read()
+      const { done, value: partialValue } = await reader.read()
       if (done) {
         break
       }
+
+      let value = partialValue
+
+      if (value[value.length - 1] !== 10) {
```

Comment:
Nit: Use a self-description variable name.
```suggestion
      const NEWLINE = '\n'.charCodeAt(0);
      if (value[value.length - 1] !== NEWLINE) {
```

We could even hoist `NEWLINE` to the top level scope.

Reply:
Done :)